src/app/component/Footer/Footer.style.scss

```diff
@@ -133,4 +133,20 @@
             }
         }
     }
+
+    &-CmsBlockWrapper {
+        .CmsBlock-Wrapper {
+            flex: 1;
+        }
+
+        .ContentWrapper .Footer-Columns {
+            padding: 0;
+        }
+
+        .Footer-Column_isNewsletter {
+            @include desktop {
+                padding-left: 100px;
```

Comment:
This is strange padding, does it work on small screens?

Reply:
On small screens the block content's layout is vertical and centered.